### PR TITLE
Update gfx942 Equality 1004_hhs_bmm sizes

### PR DESCRIPTION
**Pull request by AndySu12**

### CODE REVIEW

File: clients/include/testing_matmul.hpp [hunk 1]
```diff
@@ -1721,7 +1721,7 @@ void testing_matmul_with_bias(const Arguments& arg,
 
     // set workspace
     device_vector<unsigned char>* dWorkspace     = nullptr;
-    size_t                        workspace_size = 0;
+    size_t                        workspace_size = arg.user_allocated_workspace;
```
Comment (jichangjichang) [inline clients/include/testing_matmul.hpp]: workspace_size is only for recording the current max required workspace size.
If you want to change the max allowed workspace size, you should modify max_workspace_size as arg.user_allocated_workspace.

Comment (AndySu12) [reply clients/include/testing_matmul.hpp]: Done, thanks.

Comment (AndySu12) [reply clients/include/testing_matmul.hpp]: Assigning max_workspace_size to arg.user_allocated_workspace will make gtest fail, so still using the oldest change.

**Review by aazz44ss — DISMISSED** [review]

**Review by aazz44ss — APPROVED** [review]

File: clients/include/testing_matmul.hpp (hunk 1 above)
Comment (AndySu12) [reply clients/include/testing_matmul.hpp]: Remove workspace changes and will upload another PR to handle it.

**Comment by AndySu12** [comment]
[----------] Global test environment tear-down
[==========] 48206 tests from 13 test suites ran. (16523358 ms total)
[  PASSED  ] 48206 tests.
hipBLASLt version: 1100

**Review by alex391a — APPROVED** [review]

**Pull request merged**